File: 241008/a/main.py
"""...."""

# import math

a, b = list(map(int, input().split()))
if b == 0:
    print("Na nol' delit' nel'zya!!!")
elif a == 0:
    print(0)
elif a > 0 and b > 0:
    print(a // b)
elif a < 0 and b < 0:
    print(abs(a) // abs(b))
else:
    print(-(abs(a) // abs(b)))

File: 241008/a/main.cpp
#include <iostream>
#include <cstdint>
using namespace std;

int main(int argc, char const *argv[])
{
    int32_t A, B;
    std::cin >> A >> B;
    if (B == 0)
    {
        std::cout << "Na nol' delit' nel'zya!!!" << std::endl;
    }
    else
    {
        int Q = A / B;
        std::cout << Q << std::endl;
    }
    return 0;
}
